natss/pkg/dispatcher/dispatcher.go

```diff
@@ -30,15 +30,41 @@ import (
 	eventingduck "knative.dev/eventing/pkg/apis/duck/v1alpha1"
 	messagingv1alpha1 "knative.dev/eventing/pkg/apis/messaging/v1alpha1"
 	eventingchannels "knative.dev/eventing/pkg/channel"
+	"knative.dev/eventing/pkg/kncloudevents"
 	"knative.dev/eventing/pkg/logging"
+	"knative.dev/pkg/tracing"
 
 	"knative.dev/eventing-contrib/natss/pkg/stanutil"
-	contribchannels "knative.dev/eventing-contrib/pkg/channel"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	. "github.com/cloudevents/sdk-go/pkg/cloudevents"
 )
 
 const (
 	// maxElements defines a maximum number of outstanding re-connect requests
 	maxElements = 10
+
+	// cloudEventVersion defines the cloud events version
+	cloudEventVersion = cloudevents.VersionV1
+	// timeLayout defines the time format of cloud event payload
+	timeLayout = time.RFC3339
+	// eventType defines the type key in the cloud event payload
+	eventType = "type"
+	// eventSpecVersion defines the spec version key in the cloud event payload
+	eventSpecVersion = "specversion"
+	// eventSpecVersion defines the spec version key in the cloud event payload
+	eventTime = "time"
+	// eventDataContentType defines the content type key in the cloud event payload
+	eventDataContentType = "datacontenttype"
+	// eventId defines the event id key in the cloud event payload
+	eventId = "id"
+	// eventSource defines the event source key in the cloud event payload
+	eventSource = "source"
+	// eventDate defines the event data key in the cloud event payload
+	eventData = "data"
+
+	// tracingSpanIgnoringPath defines the tracing path to ignore
+	tracingSpanIgnoringPath = "/readyz"
```

Comment:
I'm not sure if this is the correct path.

Reply:
I think a part of this is done directly by `sharedmain` code

Reply:
I don't think is done by `sharedmain` because it's set here in `KafkaChannel`: [kafka/channel/pkg/dispatcher/dispatcher.go#L286](https://github.com/knative/eventing-contrib/blob/master/kafka/channel/pkg/dispatcher/dispatcher.go#L286)